<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>36</width>
    <height>36</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolBox" name="D">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="D_DMQ">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>300</width>
         <height>857</height>
        </rect>
       </property>
       <attribute name="label">
        <string>德国待命区</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="D_XZQ">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>300</width>
         <height>857</height>
        </rect>
       </property>
       <attribute name="label">
        <string>德国整理区</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="D_CX">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>300</width>
         <height>857</height>
        </rect>
       </property>
       <attribute name="label">
        <string>德国场下</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="dockWidget" native="true"/>
    </item>
    <item>
     <widget class="QToolBox" name="S">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="S_DMQ">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>300</width>
         <height>857</height>
        </rect>
       </property>
       <attribute name="label">
        <string>苏联待命区</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="S_XZQ">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>300</width>
         <height>857</height>
        </rect>
       </property>
       <attribute name="label">
        <string>苏联修整区</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="S_CX">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>300</width>
         <height>857</height>
        </rect>
       </property>
       <attribute name="label">
        <string>苏联场下</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>事件步骤</string>
    </property>
    <addaction name="action1"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>行动步骤</string>
    </property>
    <addaction name="action_D"/>
    <addaction name="action_S"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>轰炸步骤</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>增援步骤</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>生产步骤</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>国力步骤</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::ToolBarArea::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action1">
   <property name="text">
    <string>事件：苏联预备队</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_DTZ">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/D_TZ.png</normaloff>:/res/D_TZ.png</iconset>
   </property>
   <property name="text">
    <string>德国掷骰</string>
   </property>
   <property name="toolTip">
    <string>德国掷骰</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_STZ">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/S_TZ.png</normaloff>:/res/S_TZ.png</iconset>
   </property>
   <property name="text">
    <string>苏联掷骰</string>
   </property>
   <property name="toolTip">
    <string>苏联掷骰</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_S">
   <property name="text">
    <string>苏联先手</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_D">
   <property name="text">
    <string>德国先手</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_End">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/end.png</normaloff>:/res/end.png</iconset>
   </property>
   <property name="text">
    <string>结束环节</string>
   </property>
   <property name="toolTip">
    <string>结束环节</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
